.github/workflows/docs.yml

```
@@ -31,7 +31,7 @@ jobs:
           echo "<meta name=\"robots\" content=\"noindex\">" > header.html
 
       - name: Build docs
-        run: cargo doc --all-features --no-deps -p bevy
+        run: cargo +nightly doc --all-features --no-deps -p bevy
```

```suggestion
        run: cargo +nightly doc --all-features --no-deps -p bevy  -Zunstable-options -Zrustdoc-scrape-examples
```
from https://doc.rust-lang.org/rustdoc/scraped-examples.html